src/ShapeCrawler/Texts/TextFrame.cs

```diff
@@ -207,8 +207,8 @@ internal void Draw(SKCanvas slideCanvas, float shapeX, float shapeY)
         using var paint = new SKPaint();
         paint.Color = SKColors.Black;
         var firstPortion = this.Paragraphs.First().Portions.First();
-        paint.TextSize = (float)firstPortion.Font.Size;
-        var typeFace = SKTypeface.FromFamilyName(firstPortion.Font.LatinName);
+        paint.TextSize = (float)(firstPortion.Font?.Size ?? 11);
+        var typeFace = SKTypeface.FromFamilyName(firstPortion.Font?.LatinName ?? "Arial");
```

Comment:
If I'm not mistaken, the default font in the latest PowerPoint is "Aptos". Please change this.

Reply:
@drasmith, did you have time to check this out?

Reply:
@drasmith, could you please attach your presentation where there was an error? I will add a unit test.